tests/integ/test_tuner_multi_algo.py

```diff
@@ -63,10 +58,7 @@
 
 @pytest.fixture(scope="module")
```

Comment:
the scope of this fixture is different than other `one_p_mnist` fixtures. should the others also be module?

Reply:
yeah, that's a good call